can/generated_2021/uwrt_mars_rover_can.c

```diff
@@ -25,64 +25,77 @@
  */
 
 /**
- * This file was generated by cantools version 36.2.0 Wed Oct 13 21:03:08 2021.
+ * This file was generated by cantools version 36.7.0 Fri Jan  7 20:43:34 2022.
```

Comment:
Since there's no changes to the can structure, shouldn't the diff have no changes except for this timestamp?